bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/cb5694f4-3740-45d6-9817-44d451edb2d9.js

```diff
@@ -0,0 +1,1136 @@
+                    "password":meeting.password,
+                    "recurrence_end_times":meeting.recurrence.end_times,
+                    "recurrence_repeat_interva":meeting.recurrence.repeat_interval,
+                    "recurrence_type":meeting.recurrence.type,
+                    "settings_alternative_host":meeting.settings.alternative_hosts,
+                    "start_url":meeting.start_url,
+                    "timezone":meeting.timezone,
+                    "topic":meeting.topic,
+                    "type":Number(meeting.type),
+                    "visibility":Boolean(true)
+                }
+            }
+
+            //if recurrence type is weekly(2)
+            else if(meeting.recurrence.type == 2){
+                recurring = {
+                    "created_at":moment.utc(meeting.created_at).toDate(),
+                    "duration":Number(meeting.duration),
+                    "host_id":meeting.host_id,
+                    "host_email":meeting.host_email,
+                    "id":Number(meeting.id),
+                    "join_url":meeting.join_url,
+                    "password":meeting.password,
+                    "recurrence_end_times":meeting.recurrence.end_times,
+                    "recurrence_repeat_interva":meeting.recurrence.repeat_interval,
+                    "recurrence_type":meeting.recurrence.type,
+                    "recurrence_weekly_days":meeting.recurrence.weekly_days,
+                    "settings_alternative_host":meeting.settings.alternative_hosts,
+                    "start_url":meeting.start_url,
+                    "timezone":meeting.timezone,
+                    "topic":meeting.topic,
+                    "type":Number(meeting.type),
+                    "visibility":Boolean(true)
+                }
+            }
+
+            //if recurrence type is monthly(3)
+            else if(meeting.recurrence.type == 3){
+                recurring = {
+                    "created_at":moment.utc(meeting.created_at).toDate(),
+                    "duration":Number(meeting.duration),
+                    "host_id":meeting.host_id,
+                    "host_email":meeting.host_email,
+                    "id":Number(meeting.id),
+                    "join_url":meeting.join_url,
+                    "password":meeting.password,
+                    "recurrence_end_times":meeting.recurrence.end_times,
+                    "recurrence_repeat_interva":meeting.recurrence.repeat_interval,
+                    "recurrence_type":meeting.recurrence.type,
+                    "recurrence_monthly_day":meeting.recurrence.monthly_day,
+                    "settings_alternative_host":meeting.settings.alternative_hosts,
+                    "start_url":meeting.start_url,
+                    "timezone":meeting.timezone,
+                    "topic":meeting.topic,
+                    "type":Number(meeting.type),
+                    "visibility":Boolean(true)  
+                }

```

Comment:
Code, where you create a `recurring` object, looks very similar and repetitive

Wouldn't it be better to just first create 
```js
recurring = {
    "created_at":moment.utc(meeting.created_at).toDate(),
    "duration":Number(meeting.duration),
    "host_id":meeting.host_id,
    "host_email":meeting.host_email,
    "id":Number(meeting.id),
    "join_url":meeting.join_url,
    "password":meeting.password,
    "recurrence_end_times":meeting.recurrence.end_times,
    "recurrence_repeat_interva":meeting.recurrence.repeat_interval,
    "recurrence_type":meeting.recurrence.type,
    "settings_alternative_host":meeting.settings.alternative_hosts,
    "start_url":meeting.start_url,
    "timezone":meeting.timezone,
    "topic":meeting.topic,
    "type":Number(meeting.type),
    "visibility":Boolean(true)  
}
```
and then 
```js
 if (meeting.recurrence.type == 2) {
       recurring.recurrence_weekly_days = meeting.recurrence.weekly_days
} else (meeting.recurrence.type == 3) {
   ...
}
```